test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("Horizontal pod autoscaling (scale resource: CPU) [Skipped]", f
 	titleUp := "Should scale from 1 pod to 3 pods and from 3 to 5"
 	titleDown := "Should scale from 5 pods to 3 pods and from 3 to 1"
 
-	Describe("Deployment", func() {
+	Describe("Deployment [Feature:Deployment]", func() {
```

Comment:
Actually, @ixdy, if this is all that's blocking, can we call this "reflecting reality" right now and move on?  My next action item for #10548 is to audit all of the `[Feature:...]` tests and work with teams to get them running in the right places for v1.2.

This change is blocking getting `-serial` jobs in place, which will dramatically speed up the submit queue blockages.